cmd/api/handler/comment.go

```diff
@@ -0,0 +1,102 @@
+package handler
+
+import (
+	"context"
+	"github.com/Tiktok-Lite/kotkit/cmd/api/rpc"
+	"github.com/Tiktok-Lite/kotkit/internal/response"
+	"github.com/Tiktok-Lite/kotkit/kitex_gen/comment"
+	"github.com/Tiktok-Lite/kotkit/pkg/log"
+	"github.com/cloudwego/hertz/pkg/app"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func CommentList(ctx context.Context, c *app.RequestContext) {
+	logger := log.Logger()
+
+	videoID := c.Query("video_id")
+	token := c.Query("token")
```

Comment:
```suggestion
	token := c.Query("token")
	if token == ""
```
在http层进行一下参数非空校验会不会更合适？